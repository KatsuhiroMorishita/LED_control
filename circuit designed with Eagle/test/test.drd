%
M48
M72
T01C0.0315
T02C0.0591
T03C0.1024
%
T01
X2404Y3590
X3191Y3590
X3979Y3590
X4766Y3590
X5160Y4590
X4372Y4590
X3585Y4590
X2797Y4590
T02
X583Y4590
X6981Y4590
T03
X868Y2684
X6695Y2684
M30
