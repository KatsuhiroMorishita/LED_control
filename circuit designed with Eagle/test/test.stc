G75*
%MOIN*%
%OFA0B0*%
%FSLAX24Y24*%
%IPPOS*%
%LPD*%
%AMOC8*
5,1,8,0,0,1.08239X$1,22.5*
%
%ADD10C,0.1104*%
%ADD11R,0.0595X0.0595*%
%ADD12C,0.0595*%
%ADD13R,0.0966X0.0966*%
D10*
X000868Y002684D03*
X006695Y002684D03*
D11*
X002404Y003590D03*
D12*
X003191Y003590D03*
X003979Y003590D03*
X004766Y003590D03*
X005160Y004590D03*
X004372Y004590D03*
X003585Y004590D03*
X002797Y004590D03*
D13*
X000583Y004590D03*
X006981Y004590D03*
M02*
