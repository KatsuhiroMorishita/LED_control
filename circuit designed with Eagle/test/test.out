G75*
%MOIN*%
%OFA0B0*%
%FSLAX24Y24*%
%IPPOS*%
%LPD*%
%AMOC8*
5,1,8,0,0,1.08239X$1,22.5*
%
%ADD10C,0.0000*%
D10*
X000356Y002684D02*
X000358Y002729D01*
X000364Y002773D01*
X000373Y002817D01*
X000387Y002859D01*
X000404Y002900D01*
X000425Y002940D01*
X000449Y002978D01*
X000476Y003013D01*
X000506Y003046D01*
X000539Y003076D01*
X000574Y003103D01*
X000612Y003127D01*
X000652Y003148D01*
X000693Y003165D01*
X000735Y003179D01*
X000779Y003188D01*
X000823Y003194D01*
X000868Y003196D01*
X000913Y003194D01*
X000957Y003188D01*
X001001Y003179D01*
X001043Y003165D01*
X001084Y003148D01*
X001124Y003127D01*
X001162Y003103D01*
X001197Y003076D01*
X001230Y003046D01*
X001260Y003013D01*
X001287Y002978D01*
X001311Y002940D01*
X001332Y002900D01*
X001349Y002859D01*
X001363Y002817D01*
X001372Y002773D01*
X001378Y002729D01*
X001380Y002684D01*
X001378Y002639D01*
X001372Y002595D01*
X001363Y002551D01*
X001349Y002509D01*
X001332Y002468D01*
X001311Y002428D01*
X001287Y002390D01*
X001260Y002355D01*
X001230Y002322D01*
X001197Y002292D01*
X001162Y002265D01*
X001124Y002241D01*
X001084Y002220D01*
X001043Y002203D01*
X001001Y002189D01*
X000957Y002180D01*
X000913Y002174D01*
X000868Y002172D01*
X000823Y002174D01*
X000779Y002180D01*
X000735Y002189D01*
X000693Y002203D01*
X000652Y002220D01*
X000612Y002241D01*
X000574Y002265D01*
X000539Y002292D01*
X000506Y002322D01*
X000476Y002355D01*
X000449Y002390D01*
X000425Y002428D01*
X000404Y002468D01*
X000387Y002509D01*
X000373Y002551D01*
X000364Y002595D01*
X000358Y002639D01*
X000356Y002684D01*
X006183Y002684D02*
X006185Y002729D01*
X006191Y002773D01*
X006200Y002817D01*
X006214Y002859D01*
X006231Y002900D01*
X006252Y002940D01*
X006276Y002978D01*
X006303Y003013D01*
X006333Y003046D01*
X006366Y003076D01*
X006401Y003103D01*
X006439Y003127D01*
X006479Y003148D01*
X006520Y003165D01*
X006562Y003179D01*
X006606Y003188D01*
X006650Y003194D01*
X006695Y003196D01*
X006740Y003194D01*
X006784Y003188D01*
X006828Y003179D01*
X006870Y003165D01*
X006911Y003148D01*
X006951Y003127D01*
X006989Y003103D01*
X007024Y003076D01*
X007057Y003046D01*
X007087Y003013D01*
X007114Y002978D01*
X007138Y002940D01*
X007159Y002900D01*
X007176Y002859D01*
X007190Y002817D01*
X007199Y002773D01*
X007205Y002729D01*
X007207Y002684D01*
X007205Y002639D01*
X007199Y002595D01*
X007190Y002551D01*
X007176Y002509D01*
X007159Y002468D01*
X007138Y002428D01*
X007114Y002390D01*
X007087Y002355D01*
X007057Y002322D01*
X007024Y002292D01*
X006989Y002265D01*
X006951Y002241D01*
X006911Y002220D01*
X006870Y002203D01*
X006828Y002189D01*
X006784Y002180D01*
X006740Y002174D01*
X006695Y002172D01*
X006650Y002174D01*
X006606Y002180D01*
X006562Y002189D01*
X006520Y002203D01*
X006479Y002220D01*
X006439Y002241D01*
X006401Y002265D01*
X006366Y002292D01*
X006333Y002322D01*
X006303Y002355D01*
X006276Y002390D01*
X006252Y002428D01*
X006231Y002468D01*
X006214Y002509D01*
X006200Y002551D01*
X006191Y002595D01*
X006185Y002639D01*
X006183Y002684D01*
M02*
